colossalai/auto_parallel/meta_profiler/metainfo.py

```diff
@@ -99,14 +102,14 @@ def compute_metainfo(self):
             # module
             meta_func = meta_register.get(self._target.__class__)
 
-            # check whether the target in the module list that we don't need to save activation
-            self.save_fwd_in = self._target.__class__ not in NO_SAVE_ACTIVATION
+            # check whether the target in the list that we don't need to save activation
+            save_fwd_in = self._target.__class__ not in NO_SAVE_ACTIVATION
         except:
```

Comment:
This try-except seems dangerous.
You might use if-else instead, and raise exceptions with error messages